conan/tools/qbs/qbs.py

```diff
@@ -36,6 +45,15 @@ def _set_project_file(self, project_file):
             raise ConanException('Qbs: could not find project file %s' % self._project_file)
 
     def add_configuration(self, name, values):
+        '''
+        Adds a build configuration for the multi-configuration build.
+        This Qbs feature is rarely needed since each conan package can contain only one
+        configuration, however might be useful when creating multiple versions of the same product
+        that should be put in the same Conan package.
+        :param name: the name of the configuration. This corresponds to the ``config`` parameter
+         of ``qbs resolve``, ``qbs build`` and ``qbs install`` commands.
```

Comment:
This generates a `/conan/conan/tools/qbs/qbs.py:docstring of conan.tools.qbs.qbs.Qbs.add_configuration:6:Unexpected indentation.` error when adding this class as automatic documentation in the docs, something we might want to do in the future, please fix such indentations, they should be at the same level that the preceding line, thanks! :)